my-diploma/src/pages/Products/api/index.js

```diff
@@ -0,0 +1,13 @@
+import { api } from "../../../../src/config/config";
+import { LOCAL_STORAGE_KEYS } from "../../../constants";
+
+export const productsPageRequest = (page) =>
+  api.get(`/products?page=${page}&limit=29`);
+
+api.interceptors.request.use((axiosConfig) => {
+  const accessToken = localStorage.getItem(LOCAL_STORAGE_KEYS.ACCESS_TOKEN);
+  if (accessToken) {
+    axiosConfig.headers.Authorization = `Bearer ${accessToken}`;
+  }
+  return axiosConfig;
+});
```

Comment:
The sense of interceptors is to write 1 time and forget about this. You write this logic every single time..